docs/rst/tutorials/core/deployment/external_locators/external_locators.rst

```diff
@@ -0,0 +1,280 @@
+
+
+Prerequisites
+--------------
+
+For accomplishing this tutorial, two available hosts with Docker and a Vulcanexus image are required.
+Please refer to the installation steps detailed in :ref:`docker_installation`.
+In addition, host's ports 11200, 11201 need to be available (these ports have been selected as an example for this tutorial, but it is up to the user to make a different choice).
+
+
+*Fast-DDS* domain participants will require to announce themselves into host's external network(s).
+Net-tools, Network Manager or similar packages need to be installed in the system in order to retrieve the corresponding IPs addresses.
+
+Understanding External Locators
+--------------------------------
+
+External locators should be seen as an extra feature over the default communication mechanisms (unicast, multicast).
+In that sense, it is possible not just to discover peers via multicast within the same LAN, but also peers in remote sub-networks over a shared external network at some higher level (WAN, WLAN, etc.).
+It is for this reason that External Locators relies on the the concept of levels of externality, which effectively map to the different nesting levels of the network setup, resulting in a sequence of external IP addresses exposed by the different nested sub-network interfaces.
+
+As described in the next section, it is possible to configure external IP locators with an associated externality index, cost, and sub-network mask within the DomainParticipant configuration.
+
+
+Enable External Locators via XML configuration files
+-----------------------------------------------------
+
+In order to define the desired External Locators configuration, an XML profile needs to be provided (see `Fast DDS XML profiles <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html>`_).
+External Locators announcement for the different `Communication establishment Phases: <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery.html>`_ Participant Discovery Phase (metatraffic, initial peers tags) and Endpoint Discovery Phase (user traffic tag), should be defined.
+
+Following with the example above, two XML configuration profiles should be provided.
+The power of External Locators is the ability to connect to nodes within external networks while still being discovered in the local network.
+Due to this reasoning, multicast discovery will be used for the second container on the first host (consequently, there is no need to provide any additional XML profile).
+The two resultant XML configurations are detailed below:
+
+.. note::
+
+    Note that the container network itself does not create another level of externality in this case, as it is bridged with the host network.
+
+.. tabs::
+
+    .. tab:: HOST 1
+
+        .. tabs::
+
+            .. tab:: CONTAINER 1
+
+                .. code-block:: xml
```

Comment:
I'd move all the XML files to several resource files (like in #86) and include them with literal include.